sdks/java/io/mongodb/src/test/java/org/apache/beam/sdk/io/mongodb/MongoDBIOIT.java

```diff
@@ -57,75 +59,88 @@
  *  -DintegrationTestRunner=direct
  * </pre>
  *
- * <p>Please see 'build_rules.gradle' file for instructions regarding
- * running this test using Beam performance testing framework.</p>
+ * <p>Please see 'build_rules.gradle' file for instructions regarding running this test using Beam
+ * performance testing framework.
  */
 @RunWith(JUnit4.class)
 public class MongoDBIOIT {
 
-  private static final Map<Integer, String> EXPECTED_HASHES = ImmutableMap.of(
-    1000, "75a0d5803418444e76ae5b421662764c",
-    100_000, "3bc762dc1c291904e3c7f577774c6276",
-    10_000_000, "e5e0503902018c83e8c8977ef437feba"
-  );
+  /** MongoDBIOIT options. */
+  public interface MongoDBPipelineOptions extends IOTestPipelineOptions {
+    /* MongoDB */
```

Comment:
I think we can delete this comment because there are only MongoDB related options

Reply:
done